src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```
@@ -174,6 +174,21 @@ Copyright (c) .NET Foundation. All rights reserved.
       />
   </Target>
 
+  <!--
+    Trigger a warning if targeting a lower version of .NET Standard than is recommended
+  -->
+  <Target Name="_CheckForNotRecommendedNETStandardVersion" BeforeTargets="_CheckForInvalidConfigurationAndPlatform;Restore;CollectPackageReferences"
+          Condition="$([MSBuild]::VersionLessThan($(TargetFrameworkVersion), '2.0')) and '$(TargetFrameworkIdentifier)' == '.NETStandard'" >
```

I would swap these two conditions in case TFV is undefined (for whatever reason). That could make the msbuild intrinsic functional call fail.

```suggestion
          Condition="$(TargetFrameworkIdentifier)' == '.NETStandard' and $([MSBuild]::VersionLessThan($(TargetFrameworkVersion), '2.0'))" >
```